templates/ingress.yaml

```diff
@@ -68,8 +85,14 @@ metadata:
   {{- if .Values.ingress.flowerAnnotations }}
   {{ $ctx := . }}
   annotations:
-  {{- range $k, $v := .Values.ingress.flowerAnnotations }}
-    {{ $k | quote }}: {{ tpl $v $ctx | toYaml | indent 4 }}
+  {{- if .Values.authSidecar.enabled  }}
+    {{- range $key, $value := .Values.authSidecar.annotations}}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
```

Comment:
If we aren't passing the values through `tpl`, can we just so `{{ toYaml .Values.authSidecar.annotations | nindent x }}` instead of explicitly iterating over them?

Reply:
changes applied as per the recommendation and tested.